### PR TITLE
Added labels to views

### PR DESCRIPTION
**Pull request by ruthmd**

### CODE REVIEW

**Comment by ranjankol** [comment]
@ruthmd ,
Please find the comment added

File: j2v/generation/result_writer.py [hunk 1]
```diff
@@ -28,8 +28,8 @@ def get_view_str(self, views_dimensions_expr):
             source_table = ""
             if view == self.table_alias:
                 source_table = """\n  sql_table_name: {sql_table} ;;""".format(sql_table=self.sql_table_name)
-
-            views_out.append(lt.view_start_str_template.format(name=view, base_table=source_table))
+            nice_label = " ".join(view.split("_")).title()
```
[the remark this answers is not in the document]
Comment (ruthmd) [reply j2v/generation/result_writer.py]: No, this is a part of feature that i've added!

[the remark this answers is not in the document]
Comment (ranjankol) [reply j2v/generation/result_writer.py]: @ruthmd ,
If there is no value against `title()` how it behaves?

[the remark this answers is not in the document]
Comment (ruthmd) [reply j2v/generation/result_writer.py]: It is a python inbuilt  function.
https://docs.python.org/3/library/stdtypes.html#str.title

**Pull request merged**